src/caffe/layers/cudnn_conv_layer.cpp

```diff
@@ -170,26 +172,19 @@ void CuDNNConvolutionLayer<Dtype>::Reshape(
   if (use_algo_seeker_) {
     size_t workspace_limit_bytes, total_memory;
     GPUMemory::GetInfo(&workspace_limit_bytes, &total_memory);
-    // FindEx: A workspace of size workspace_bytes is allocated for FindEx.
+    // FindEx: A workspace of size WORKSPACE_SIZE is allocated for FindEx.
     //         Besides, workspace, a buffer is allocated for the output of
     //         FindEx-backward-filter. The size of buffer is as big as weights.
-    // Get: workspace_bytes is only used as a workspace limit by Get.
+    // Get: WORKSPACE_SIZE is only used as a workspace limit by Get.
     //      (no allocation happens before Get or by Get).
     size_t workspace_bytes;
     if (use_modest_workspace_) {
       // In iteration 0, use a small amount of memory in order to leave
       // most of memory for allocating layer blobs.
-      // TODO: Read 8*1024*1024 from a data member variable.
-      workspace_bytes = 8*1024*1024;
+      workspace_bytes = INITIAL_WORKSPACE_SIZE;
     } else {
-      // Use 90% of available memory.
-      // Using all of memory may result in failure of workspace.reserve.
-      // TODO: Since 90% of memory might be too large, we can allocate
```

Comment:
I meant this:
```
      // TODO: Since 90% of memory might be too large, we can allocate
      //       exactly how much FindEx needs by taking the maximum
      //       workspace among all algorithms (requires an initial call
      //       to FindEx with workspace size 0).
```